.travis.yml

```diff
@@ -6,7 +6,9 @@ matrix:
       jdk: openjdk8
 
 go:
-  - "1.12"
+  - "1.12.x"
```

Comment:
just one go version is enough.

Reply:
the dependend libs may need diffirent version. multi version testing may help us find the errors caused by lib-version